assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -102,6 +157,26 @@ export function useHelpCenter() {
     [store.actions]
   );
 
+  useEffect(() => {
+    let isMounted = true;
+    getCurrentUser()
+      .then(({ meta }) => {
+        isMounted &&
+          actions.hydrateReadTipsSuccess({
+            readTips: meta.web_stories_onboarding ?? {},
+          });
+      })
+      .catch(actions.persitingReadTipsError);
+    return () => (isMounted = false);
+  }, [actions, getCurrentUser]);
```

Comment:
Don't think this is required, it is loaded [here](https://github.com/google/web-stories-wp/blob/3298bd4d0a44a7bc3ae98d897db65b90ad29bf00/assets/src/edit-story/app/currentUser/currentUserProvider.js#L35-L39). It is even cached. Can we use this provider?